runeq/resources/stream_metadata.py

```diff
@@ -712,40 +676,48 @@ def get_stream_availability_dataframe(
                 from the API. Otherwise, the global StreamClient is used.
 
         """
-        params = {
-            'stream_id': self.ids(),
-            'start_time': start_time,
-            'end_time': end_time,
-            'resolution': resolution,
-            'batch_operation': batch_operation,
-            'format': 'csv',
-            'limit': limit,
-            'page_token': page_token,
-            'timestamp': timestamp,
-            'timezone': timezone,
-            'client': stream_client,
-        }
+        responses = get_stream_availability(
+            stream_ids=list(self.ids()),
```

Comment:
nitpick: is `list()` needed here?

Reply:
only to make the type-hints happy... `self.ids()` iterates over the IDs, but `get_stream_availability` expects a list. I could loosen that in the `get_stream_availability` typing?

Reply:
Didn't realize `self.ids()` was returning an iterator, thought it was returning a list itself. This is totally good!